README.md

```diff
@@ -1,273 +1,88 @@
-
-To run this example, clone the TorchServe repository:
-
-```bash
-git clone https://github.com/pytorch/serve.git
-```
-
-Then run the following steps from the parent directory of the root of the repository.
-For example, if you cloned the repository into `/home/my_path/serve`, run the steps from `/home/my_path`.
-
-### Store a Model
-
-To serve a model with TorchServe, first archive the model as a MAR file. You can use the model archiver to package a model.
-You can also create model stores to store your archived models.
-
-1. Create a directory to store your models.
-
-    ```bash
-    mkdir model_store
-    ```
-
-1. Download a trained model.
-
-    ```bash
-    wget https://download.pytorch.org/models/densenet161-8d451a50.pth
-    ```
-
-1. Archive the model by using the model archiver. The `extra-files` param uses a file from the `TorchServe` repo, so update the path if necessary.
-
-    ```bash
-    torch-model-archiver --model-name densenet161 --version 1.0 --model-file ./serve/examples/image_classifier/densenet_161/model.py --serialized-file densenet161-8d451a50.pth --export-path model_store --extra-files ./serve/examples/image_classifier/index_to_name.json --handler image_classifier
-    ```
-
-For more information about the model archiver, see [Torch Model archiver for TorchServe](model-archiver/README.md)
-
-### Start TorchServe to serve the model
-
-After you archive and store the model, use the `torchserve` command to serve the model.
-
-```bash
-torchserve --start --ncs --model-store model_store --models densenet161.mar
-```
-
-After you execute the `torchserve` command above, TorchServe runs on your host, listening for inference requests.
-
-**Note**: If you specify model(s) when you run TorchServe, it automatically scales backend workers to the number equal to available vCPUs (if you run on a CPU instance) or to the number of available GPUs (if you run on a GPU instance). In case of powerful hosts with a lot of compute resources (vCPUs or GPUs), this start up and autoscaling process might take considerable time. If you want to minimize TorchServe start up time you should avoid registering and scaling the model during start up time and move that to a later point by using corresponding [Management API](docs/management_api.md#register-a-model), which allows finer grain control of the resources that are allocated for any particular model).
-
-### Get predictions from a model
-
-To test the model server, send a request to the server's `predictions` API. TorchServe supports all [inference](docs/inference_api.md) and [management](docs/management_api.md) api's through both [gRPC](docs/grpc_api.md) and [HTTP/REST](docs/rest_api.md).
-
-#### Using GRPC APIs through python client
-
- - Install grpc python dependencies :
- 
 ```bash
-pip install -U grpcio protobuf grpcio-tools
+curl http://127.0.0.1:8080/predictions/bert -T input.txt
```

Comment:
@msaroufim are we keeping this here as for showing the ease of use?

Reply:
Yeah do you think it does a good job at showcasing ease of use? I also thought we could showcase the requests library but I figured everyone uses curl